ocaml-lsp-server/src/doc_to_md.ml

```diff
@@ -124,12 +124,12 @@ let rec nestable_block_element_to_block
     let meta = loc_to_meta location in
     Block.List (l, meta)
   | Odoc_parser.Loc.
-      { value = `Code_block (metadata, { value = code; location = _code_loc })
+      { value = `Code_block (metadata, { value = code; location = code_loc })
       ; location
       } ->
     let info_string =
       match metadata with
-      | None -> None
+      | None -> Some ("ocaml", loc_to_meta code_loc)
```

Comment:
I don't think that's right, can we assume that every ocamldoc code block is in ocaml?

Reply:
I assume that we default to ocaml when lang header is missing. See: https://ocaml.github.io/odoc/odoc_for_authors.html#ocaml_code_blocks

Also, I've added an explicit test case for this